modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPreloader.java

```diff
@@ -144,6 +148,16 @@ private IgniteCheckedException stopError() {
         return new NodeStoppingException("Operation has been cancelled (cache or node is stopping).");
     }
 
+    /**
+     * Returns a boolean value of flag, getting though JVM property.
+     * @see IGNITE_DISABLE_REBALANCING_CANCELLATION_OPTIMIZATION
+     *
+     * @return Rebalance cancellation optimization flag.
+     */
+    public boolean isDisableRebalancingCancellationOptimization() {
```

Comment:
Against code style.
Method must be named disableRebalancingCancellationOptimization()